.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+Fixes #
```

Comment:
Could we move this to the end of the template?

Reply:
sure!

Reply:
Seems like it moved your comment as well :P